src/site/content/en/blog/ar-hit-test/index.md

```diff
@@ -4,14 +4,14 @@ subhead: The Hit Test API lets you position virtual items in a real-world view.
 authors:
   - joemedley
 date: 2020-02-13
-updated: 2020-03-19
+updated: 2020-04-20
```

Comment:
I'd recommend not changing the `upated` field for any of theses posts because it will show up on the post itself and make it seem like content was updated.

Reply:
I think these were revved by our precommit hook. You can bypass it with `git commit --no-verify`. I'm updating the instructions in https://github.com/GoogleChrome/web.dev/blob/master/.huskyrc to mention this.